Gaijinizer patch/Troops.txt

```diff
@@ -10679,30 +10690,30 @@ I laughed with the beast male and below.#MTLed
 ていうか、吠え方に似てる部分があるとは思ったけど
  まさか本当にバーゲストだったんだ……
 > CONTEXT: Troops/283/pages/0/48/Dialogue/kazuya
-.......... As usual, personality is bad ...
-In other words, I thought that there is a part
-that resembles how to lick. No way it was really
-a barghest ......#MTLed
+She's a nasty piece of work, as expected...
+But wait a minute, you both sound kind of similar
+to each other, don't tell me that was Barghest's
+previous form...#RonIffy
 > END STRING
```

Comment:
rewording for flow:
>_**But wait a minute, you look kind of similar
to that... don't tell me that was Barghest's
previous form...**_